srcpkgs/dracut/template

```diff
@@ -58,10 +62,14 @@ dracut-uefi_package() {
 	depends="binutils gummiboot-efistub ${sourcepkg}>=${version}_${revision}"
 	short_desc+=" - UEFI bundle hook"
 	conf_files="/etc/default/dracut-uefi-hook"
+	alternatives="
+	 initramfs-hooks:/usr/libexec/dracut/kernel-uefi-post-install:/etc/kernel.d/post-install/20-initramfs
+	 initramfs-hooks:/usr/libexec/dracut/kernel-uefi-post-remove:/etc/kernel.d/post-remove/20-initramfs
```

Comment:
Are UEFI generators really an "alternative" to traditional initramfs generators? Do they conflict? And what would happen in this case, if someone runs `xbps-install dracut-uefi`? They would have to switch the alternative after, right?

Reply:
Yes and no. I was torn about adding an alternative here because it doesn't conflict in the same way as mkinitcpio/dracut, but presumably somebody interested in spitting out a UEFI bundle doesn't care about a separate initramfs in `/boot`. We could skip the alternative here and just install the hooks as they are now, or a user who wishes for both a separate initramfs and a UEFI bundle could manually link to the new hooks in `/usr/libexec`.

Motivation for this comes from https://github.com/void-linux/void-packages/pull/39282.